nixos/tests/kernel-signed-modules.nix

```diff
@@ -0,0 +1,23 @@
+import ./make-test.nix ({ pkgs, ...} : {
```

Comment:
I think we want this file in release.nix so that it is run by Hydra.

Reply:
I just pushed a commit adding it to release.nix.